### PR TITLE
rename sql unit test

### LINKED ISSUE #468: Rename sql/db_test.go
Per @hungry1526's suggestion, the file names `sql/database_test.go` and `sql/db_test.go` look confusing. Since `sql/db_test.go` mostly contains `TestMain`, shall we name it to something else. What is the convention of Go in naming the test entrance?

### PR DESCRIPTION
**Pull request by Yancey0623**
Fixed #468

### CODE REVIEW

File: sql/sql_test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	assertNoErr(testdata.Popularize(testDB.DB, testdata.IrisSQL))
+	assertNoErr(testdata.Popularize(testDB.DB, testdata.ChurnSQL))
+}
+
+func testHiveSQL() {
+	var e error	
+	// NOTE: sample dataset is written in
+	// https://github.com/sql-machine-learning/gohive/blob/develop/docker/entrypoint.sh#L123
+	testDB, e = Open("hive://root:root@localhost:10000/churn")
+	defer testDB.Close()
```
Comment (tonyyang-svail) [inline sql/sql_test.go]: `testDB` is used in other tests such as `verifier_test.go` and `executor_test.go`. So we shouldn't defer in this function. :)

Comment (Yancey0623) [reply sql/sql_test.go]: Thx, moved the defer to the main function.

File: sql/sql_test.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	assertNoErr(testdata.Popularize(testDB.DB, testdata.IrisHiveSQL))
+	assertNoErr(testdata.Popularize(testDB.DB, testdata.ChurnHiveSQL))
+	return testDB
+}
+
+func TestMain(m *testing.M) {
+	dbms := getEnv("SQLFLOW_TEST_DB", "mysql")
+	switch dbms {
+	case "sqlite3":
+		testDB := testSQLiteSQL()
```
Comment (tonyyang-svail) [inline sql/sql_test.go:79]: `:=` => `=`. I made the same mistake several days ago... >_<

Comment (Yancey0623) [reply sql/sql_test.go:79]: emmm...missing the global var -.-

**Review by tonyyang-svail — APPROVED** [review]
LGTM.

**Pull request merged**